src/frontends/pytorch/src/op/search_sorted.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/search_sorted.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_search_sorted(const NodeContext& context) {
+    num_inputs_check(context, 2, 5);
+    Output<Node> sorted;
+    Output<Node> values;
+    std::tie(sorted, values) = get_inputs_with_promoted_types(context, 0, 1);
+    const bool out_int32 = context.const_input<bool>(2);
+    PYTORCH_OP_CONVERSION_CHECK(out_int32 == false, "aten::searchsorted(out_int32=true) unsupported");
```

Comment:
Is it not just the output type of the operation? Can it be supported by just inserting `Convert` to i32?

Reply:
It is a part of aten::searchsorted API and it is(that flag) not supported at the moment as it is not needed.